### PR TITLE
rustdesk 1.2.3-2

### PR DESCRIPTION
**Pull request by aivus**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/r/rustdesk.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
   livecheck do
     url :url
+    regex(/(\d+(?:\.\d+(\-\d+)?)+)/)
```
Comment (aivus) [inline Casks/r/rustdesk.rb:16]: Version format like `1.2.3-2` is used by this cask. Added regexp (based on the default) with addition.

File: Casks/r/rustdesk.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 cask "rustdesk" do
   arch arm: "aarch64", intel: "x86_64"
 
-  version "1.2.3"
+  version "1.2.3-2"
```
Comment (aivus) [inline Casks/r/rustdesk.rb:4]: As for the hashes, it seems that `rustdesk` updated existing releases and use same binary for `1.2.3`, `1.2.3-1` and `1.2.3-2`:
```
➜  ~   shasum -a 256 rustdesk-1.2.*                                                                                                     
63f1f5bbe2a308495be039604886adc83937bc36e97b09d4fd9b675ee108ee09  rustdesk-1.2.3-1-x86_64.dmg
63f1f5bbe2a308495be039604886adc83937bc36e97b09d4fd9b675ee108ee09  rustdesk-1.2.3-2-x86_64.dmg
63f1f5bbe2a308495be039604886adc83937bc36e97b09d4fd9b675ee108ee09  rustdesk-1.2.3-x86_64.dmg
...
647bc014238086d7a73a3d8f1543a1a559b5240f1796e1311a2142be0b57152e  rustdesk-1.2.3-1-aarch64.dmg
647bc014238086d7a73a3d8f1543a1a559b5240f1796e1311a2142be0b57152e  rustdesk-1.2.3-2-aarch64.dmg
647bc014238086d7a73a3d8f1543a1a559b5240f1796e1311a2142be0b57152e  rustdesk-1.2.3-aarch64.dmg

File: Casks/r/rustdesk.rb (hunk 1 above)
Comment (bevanjkay) [reply Casks/r/rustdesk.rb:16]:
```suggestion
    regex(/^v?(\d+(?:[.-]\d+)+)$)/i)
```

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @aivus.

**Pull request merged**